add install info on desktop apps (#354)
actualbudget/docs: docs/install/index.md

# Installing Actual

The Actual Budget project is made up of two parts: the client and the server.  A server is not required for Actual to function but it is strongly recommended as it provides increased functionality. Below is a table of features of Actual and if those features work with just a client or if a server is needed. 

| Feature | Without Server | Needs Server |
|:-|:-:|:-:|
| Import transactions from files | X | |
| All budgeting features (budgets, reports, schedules, etc) | X | |
| Import or Export budget files | X | |
| Use Actual on a mobile device | | X[^1] |
| Use Actual in a web browser | | X[^1] |
| Sync budget between devices| | X |
| Use bank syncing (GoCardless or SimpleFIN)| | X |
| Use the Actual API | | X |

> The standard way of using Actual is to setup a personal server and use a web browser for the application.  For quick testing or getting to know Actual before setting up a server, the [demo](https://demo.actualbudget.org) or the [desktop applications](https://github.com/actualbudget/actual/releases) are a good place to start.

Reviewer: When used as a verb, it is "set up" and not "setup".
```suggestion
The standard way of using Actual is to set up a personal server and use a web browser for the application.  For quick testing or getting to know Actual before setting up a server, the [demo](https://demo.actualbudget.org) or a [desktop application](https://github.com/actualbudget/actual/releases) are a good place to start.
```

> ## Server Optional Client Options

Reviewer: ```suggestion
## Server-Optional Client Options
```

> Desktop applications are available for Windows, Mac, and Linux. These can be [downloaded from GitHub](https://github.com/actualbudget/actual/releases).  The desktop apps have the following benefits:
> * Can connect to a server (this gives them the ability to use the server based features)
> * Automated backups
> * Offline use is ready out of the box 

Reviewer: ```suggestion
Desktop applications are available for Windows, Mac, and Linux. These can be [downloaded from GitHub](https://github.com/actualbudget/actual/releases).  The desktop apps have the following benefits:
* Can connect to a server (this gives them the ability to use the server-based features)
* Automated backups
* Offline use is ready out of the box 
```

> ## Server Based Client Options

Reviewer: ```suggestion
## Server-Based Client Options
```

> The server provides a web based version of Actual.  This web app can be used in a browser as a standard web page to use view and edit your budget.  The web page can also be installed to your device.  For mobile devices an installed web page will work offline.

Reviewer: ```suggestion
The server provides a web-based version of Actual.  This web app can be used in a browser as a standard web page to view and edit your budget.  The web page can also be installed on your device.  For mobile devices, an installed web page will work offline.
```

- [Synology NAS](https://mariushosting.com/how-to-install-actual-on-your-synology-nas/)
- [Home Assistant](https://github.com/sztupy/hassio-actualbudget/blob/main/README.md)

> [^1]: You technically don't need a server instance for this. You need to run either a web based client or a server, but a server is the same effort.
> [^2]: A portion of the cost to host on PikaPods is donated to the Actual Budget Project.  With that said, PikaPods is a very simple, and cost effective way, to host your server.

Reviewer: ```suggestion
[^1]: You technically don't need a server instance for this. You need to run either a web-based client or a server, but a server is the same effort.
[^2]: A portion of the cost to host on PikaPods is donated to the Actual Budget Project.  With that said, PikaPods is a very simple, and cost-effective way, to host your server.
```
